services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_services.py

```diff
@@ -13,6 +15,15 @@
     get_services_client,
 )
 
+from simcore_service_director_v2.modules.dynamic_sidecar.entrypoint import (
+    start_dynamic_sidecar_stack_for_service,
+    get_dynamic_sidecar_stack_status,
```

Comment:
We normally use the files under ``api/routes`` to bind routes with handler functions . Since FastAPI does such a good job in abstracting requests -> model conversion, we also include most of the implementation code in these files. Is there any reason why you created an *extra* redirection and moved the latter to ``modules.dynamic_sidecar.entrypoint``??

Reply:
With the exception of the `start_dynamic_sidecar_stack_for_service` all other functions have been moved.
This module was simply born like this. It makes more sense to have these functions located like they currently are in my opinion. As the `dynamic_sidecar` was meant to be a self contained module and it was later attached to the API.
I would keep it as it is in the current iteration.